src/modules/musig/musig.md

```diff
@@ -0,0 +1,196 @@
+MuSig - Rogue-Key-Resistant Multisignatures Module
+===========================
+
+This module implements the MuSig [1] multisignature scheme. The majority of
+the module is an API designed to be used by signing or auditing participants
+in a multisignature scheme. This involves a somewhat complex state machine
+and significant effort has been taken to prevent accidental misuse of the
+API in ways that could lead to accidental signatures or loss of key material.
+
+# Theory
+
```

Comment:
This paragraph uses "signers" and "participants" for the same thing. That's not necessarily bad it actually reminded me that there can be participants who are not signers (just listening -- and the code supports that). So maybe we should make that clearer.

Reply:
ah I see that distinction is made below, so yes, I think it should be made here too

Reply:
I changed this paragraph.